pages/reference.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="main-content" class="text-wrapper">
+  <div id="main-content" class="text-wrapper" style="width:100%">
```

Comment:
what is this doing? I removed the style in Firefox and didn't see any difference. It might be affecting resolutions I'm not aware of so thought I'd check this with you. Thanks Ruby

Reply:
I can't see a difference either sorry not sure why I put that there. Will remove that.